pkg/chrootarchive/chroot_linux.go

```diff
@@ -57,13 +62,6 @@ func chroot(path string) (err error) {
 				err = errCleanup
 			}
 		}
-
-		if errCleanup := syscall.Unmount("/", syscall.MNT_DETACH); errCleanup != nil {
-			if err == nil {
-				err = fmt.Errorf("error unmounting root: %v", errCleanup)
-			}
-			return
-		}
```

Comment:
Should we put a comment here / somewhere to explain why we don't cleanup? Trying to prevent future contributors from opening a PR that adds this back and we forgot about why we're not doing that